src/main/java/com/questhelper/quests/gertrudescat/GertrudesCat.java

```diff
@@ -0,0 +1,219 @@
+package com.questhelper.quests.gertrudescat;
+
+import com.questhelper.QuestDescriptor;
+import com.questhelper.QuestHelperQuest;
+import com.questhelper.Zone;
+import com.questhelper.panel.PanelDetails;
+import com.questhelper.questhelpers.BasicQuestHelper;
+import com.questhelper.requirements.ItemRequirement;
+import com.questhelper.steps.ConditionalStep;
+import com.questhelper.steps.NpcStep;
+import com.questhelper.steps.ObjectStep;
+import com.questhelper.steps.QuestStep;
+import com.questhelper.steps.conditional.ConditionForStep;
+import com.questhelper.steps.conditional.Conditions;
+import com.questhelper.steps.conditional.ItemCondition;
+import com.questhelper.steps.conditional.ItemRequirementCondition;
+import com.questhelper.steps.conditional.ZoneCondition;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+import net.runelite.api.IconID;
+import net.runelite.api.ItemID;
+import net.runelite.api.NpcID;
+import net.runelite.api.ObjectID;
+import net.runelite.api.coords.WorldPoint;
+
+@QuestDescriptor(
+	quest = QuestHelperQuest.GERTRUDES_CAT
+)
+public class GertrudesCat extends BasicQuestHelper
+{
+	private ItemRequirement bucketOfMilk, coins, seasonedSardine;
+
+	private QuestStep talkToGertrude, talkToChildren,
+		gertrudesCat, gertrudesCat2, searchNearbyCrates, giveKittenToFluffy,
+		finishQuest;
+
+	private ConditionForStep isUpstairsLumberyard, hasFluffsKitten;
+
+	@Override
+	public Map<Integer, QuestStep> loadSteps()
+	{
+		setupItemRequirements();
+		setupZones();
+		setupConditions();
+
+		return getSteps();
+	}
+
+	private Map<Integer, QuestStep> getSteps()
+	{
+		Map<Integer, QuestStep> steps = new HashMap<>();
+
+		steps.put(0, talkToGertrude = getTalkToGertrude());
+		steps.put(1, talkToChildren = getTalkToChildren());
```

Comment:
Rather than recommending a seasoned sardine, it'd be good to recommend a raw sardine and have 2 steps for getting a doogle leaf and using the doogle on the sardine. It'd make it a bit more ironman friendly, and avoid players paying extra for about 5 extra seconds.

Reply:
Should be good now?